src/Tizen.NUI.Components/Controls/Navigation/DialogPage.cs

```diff
@@ -198,7 +198,26 @@ public bool EnableScrim
         /// Indicates to dismiss dialog by touching on scrim.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public bool EnableDismissOnScrim { get; set; }
+        public bool EnableDismissOnScrim { get; set; } = true;
```

Comment:
How about `DismissOnScrim`?
`EnableDismissOnScrim` seems to be a name of method.